src/network/rtnet/rtsp/serializer.cpp

```diff
@@ -0,0 +1,33 @@
+#include "serializer.hpp"
+
+
+namespace shar::rtsp {
+
+Serializer::Serializer(char * buffer, std::size_t size)
+  : m_buffer_begin(buffer), m_size(size){}
+
+bool Serializer::append_string(const char * string) {
+  std::size_t str_len = std::strlen(string);
```

Comment:
`strlen` is O(n) - not good. `append_string` could be implemented the same way as `append_number`, which would allow to remove `strlen` call.